### PR TITLE
Fix pylint install on Travis CI

### PR DESCRIPTION
**Pull request by LaszloLango**
Followup fix after #1602

JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,6 +49,13 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]] && [[ "$INSTALL_QEMU_ARM" == "yes" ]]; then tools/apt-get-install-qemu-arm.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then tools/brew-install-deps.sh; fi
   - if [[ "$OPTS" == "--test262" ]]; then sudo timedatectl set-timezone America/Los_Angeles; fi
+  - if [[ "$OPTS" == *"--check-pylint"* ]];
+    then
+      git clone https://github.com/PyCQA/pylint.git -b pylint-1.6.5 "$PWD/third-party/pylint";
```
Comment (galpeter) [inline .travis.yml]: What if we do the following? ` pip install pylint==1.6.5`
Then yo don't need to manually call clone/setup.

Comment (LaszloLango) [inline .travis.yml]: I'll check it. Thanks for the tip.

Comment (LaszloLango) [inline .travis.yml]: It works as expected and much simpler, so I've updated the PR. Please check.

**Comment by galpeter** [comment]
lgtm

**Pull request merged**